text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,116 @@
+
+**Note:** In cases where some cores remain unsold in the market, all buyers are obligated to pay the `RESERVE_PRICE`.
+
+#### Renewal Period (7 days)
+
+As the `RENEWAL_PERIOD` commences, all current tenants are granted the opportunity to renew their cores at a slight discount of `MARKET_PRICE * RENEWAL_DISCOUNT` (for instance, 10%). This provision affords marginal benefits to existing tenants, balancing out the non-transferability aspect of renewals.
+
+At the end of the period, all available cores are allocated to the current tenants who have opted for renewal and the participants who placed bids during the market period. If the demand for cores exceeds supply, the cores left unclaimed from renewals may be awarded to bidders who placed their bids early in the auction, thereby subtly incentivizing early participation. If the supply exceeds the demand, all unsold cores are transferred to the Instantanous Market.
+
+#### Reserve Price Adjustment
+
+After all cores are allocated, the `RESERVE_PRICE` is adjusted following the process described in RFC-1 and serves as baseline price in the next `BULK_PERIOD`. 
+
+**Note:** The particular price curve is outside the scope of the proposal. The `MARKET_PRICE` (as a function of `RESERVE_PRICE`), however, is able to capture higher demand very well while being capped downwards. That means, the curve that adjusts the `RESERVE_PRICE` should be more sensitive to undercapacity.
+
+#### Settlement Period (7 days)
+
+There are no explicit tasks for the coretime systemchain to perform in this period. This period is still important to allow for sufficient time for all participants to engage in secondary markets, especially to be able to allocate Coretime that starts in the upcoming `BULK_PERIOD`.
+
+#### Instantanous Market
+
+Blockspace sold through the Instantaneous Market can be viewed as a [substitute good](https://en.wikipedia.org/wiki/Substitute_good) to fixed allocated blockspace. As such, we can expect prices between the Bulk Market and the Instantaneous Market to have a close correlation, aligning closely with the `MARKET_PRICE`. Given the less flexible nature of on-demand blocks, it might be beneficial to introduce a small discount.
+
+A good approach could be to commence the Instantaneous Market with an `INSTANTANOUS_START_PRICE = MARKET_PRICE * INSTANTANOUS_DISCOUNT` (potentially 20%) at the outset of a `BULK_PERIOD`. Under the assumption of a 28-day period, the price for the first block would be set at 
```

Comment:
This may be out-of-scope. 

I think coordinating Instantaneous prices with bulk prices is difficult to do in practice, although the Coretime chain could periodically send hints for updating the base price for instantaneous cores. These would probably be better as hints, though, and treated as a nudge to the current spot price rather than a reset.

Reply:
I am only proposing to initiate the instantanous market with a price. It would quickly adapt (block-by-block) by some supply/demand function as proposed in the original RFC (via some queue system).  My general assumption is that coretime of the instantanous market is a (non-perfect) substitute, so I expect prices to be somewhat correlated. Therefore, it is reasonable to initiate the market with the market price (and some discount) and then let the price adjust dynamically block by block.

Reply:
That is still a quite strong assumption - even that instantaneous might be discounted. Perhaps instantaneous coretime is _more_ valuable. Forcibly adjusting the sell price rather than allowing it to naturally float towards its market price (which, if it is an imperfect substitute, should be in the neighborhood anyway) seems unnecessary.

Reply:
You might be right, I take the argument that the market would drive the price on the instantanous market towards the bulk market prices on its own. I'll remove it from the proposal.